core/database/foxx/api/user_router.js

```diff
@@ -1272,8 +2716,29 @@ router
 
 router
     .get("/ep/set", function (req, res) {
+        let client = undefined;
         try {
-            const client = g_lib.getUserFromClientID(req.queryParams.client);
+            client = g_lib.getUserFromClientID(req.queryParams.client);
+            console.info(
+                "Client:",
+                client?._id || "unknown",
+                "|",
+                "Correlation_ID:",
+                req.headers["x-correlation-id"],
+                "|",
+                "HTTP:",
+                "GET",
+                "|",
+                "Route:",
+                "/ep/set",
+                "|",
+                "Status:",
+                "Started",
+                "|",
+                "Desc:",
+                "Set recent end-points",
```

Comment:
It would be good to include the end points from the request if any are valid in the log messsage.